rock_paper_scissors.py

```diff
@@ -97,11 +97,14 @@ def rounds(self):
 
     @rounds.setter
     def rounds(self, value):
-        if value % 2 != 0:
-            self._rounds = value
-        else:
+        try:
+            value = int(value)
+            if value % 2 == 0 or value < 0:
+                raise ValueError
+        except ValueError:
             print('Please enter odd number of rounds.')
-            exit()
```

Comment:
Ideally I would like it to re-prompt the user into selecting the correct number of rounds. I left it as is with the program ending if exception was raised.